tasmota/my_user_config.h

```diff
@@ -470,7 +470,8 @@
   #define USE_ENHANCED_GUI_WIFI_SCAN             // Enable Wi-Fi scan output with BSSID (+0k5 code)
 //  #define USE_WEBSEND_RESPONSE                   // Enable command WebSend response message (+1k code)
 //  #define USE_WEBGETCONFIG                       // Enable restoring config from external webserver (+0k6)
-//  #define USE_GPIO_VIEWER                        // Enable GPIO Viewer to see realtime GPIO states (+6k code)
+//  #define USE_GPIO_VIEWER                        // Enable GPIO Viewer to see realtime GPIO states (+4k code)
+//    #define GV_BASE_URL           "https://thelastoutpostworkshop.github.io/microcontroller_devkit/gpio_viewer/assets/"
```

Comment:
Pls rebase your PR as this line has been removed

Reply:
done - I hope thats what you need  :-))

Reply:
I am confused ...
The "my_user_config.h" in my fork is only extended with some #define USE_QMC58...
I see the identical #define USE_GPIO_VIEWER  in both "my_user_config.h" (in my fork and in the original development in two lines :
474 ... //  #define USE_GPIO_VIEWER                        // Enable GPIO Viewer to see realtime GPIO states (+4k code)
and
1105 ... #define USE_GPIO_VIEWER                          // Enable GPIO Viewer to see realtime GPIO states (+5k6 code)
line 475 is ...
475 ... //    #define GV_SAMPLING_INTERVAL  100            // [GvSampling] milliseconds - Use Tasmota Scheduler (100) or Ticker (20..99,101..1000)
??? what can I do ???

Reply:
I  rebased my PR on dev[13.4.0.1] now 
I hope it works now

Reply:
No it doesn't. You should be able to easily see that it fails on line 475. That line is not present in dev or released branch.

So keep trying until you see for yourself that line 475 is gone.

Reply:
I'll fix it.